src/dynamics/integration_parameters.rs

```diff
@@ -221,7 +221,11 @@ impl IntegrationParameters {
     pub fn joint_cfm_coeff(&self) -> Real {
         // Compute CFM assuming a critically damped spring multiplied by the damping ratio.
         // The logic is similar to `Self::cfm_factor`.
```

Comment:
should a similar correction be applied to `cfm_factor` ?

Reply:
Yes

Reply:
also, the correct function referred to is probably `contact_cfm_factor` ?